pilot/helpers/test_cli.py

```diff
@@ -8,10 +8,9 @@
 @patch("helpers.cli.subprocess")
 def test_terminate_process_not_running(mock_subprocess, mock_os):
     terminate_process(1234, 'not running')
-    if platform.system() == 'Windows':
-        mock_subprocess.run.assert_called_once_with(["taskkill", "/F", "/T", "/PID", "1234"])
-    else:
-        mock_os.killpg.assert_called_once_with(1234, 9)
+
+    mock_subprocess.run.assert_not_called()

```

Comment:
Note that if we'd move to use `psutil.Process.kill` (see other comment) this test would need changing to mock that instead, and to check it was called.